### PR TITLE
Remove `Device.activeCall`.

### PR DESCRIPTION
**Pull request by mhuynh5757**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### JIRA link(s):

- [VBLOCKS-145](https://issues.corp.twilio.com/browse/VBLOCKS-145)

### Description

Remove `Device.activeCall` public class member.

### Before review
* [x] Updated CHANGELOG.md if necessary
* [x] Added unit tests if necessary
* [x] Updated affected documentation
* [x] Verified locally with `npm test`
* [x] Manually sanity tested running locally
* [x] Ready for review

### Before merge
* [ ] Got one or more +1s
* [ ] Squashed erroneous commits if necessary
* [ ] Re-tested if necessary

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,32 @@
+2.0.0 (In Progress) - Release
+===================
+
+Breaking API Changes
+--------------------
+
+### Active call no longer accessible through the Device object.
+
+To future-proof the SDK, `Device.activeCall` is no longer available. Instead,
```
Comment (charliesantos) [inline CHANGELOG.md:9]: Do we really need to mention "To future-proof the SDK"? I think we can remove this, it doesn't tell me why this is future proofing as the customer.

**Review by charliesantos — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,32 @@
...
+const token = '...';
+const options = { ... };
+const device = new Device(token, options);
+
+await device.register();
+
+const activeCall = await device.connect();
+```
+
 2.0.0 (Apr 30, 2021) - Pilot
```
Comment (idelgado) [inline CHANGELOG.md:23]: I would expect a Pilot release to have a specifier at the end to indicate as much. Did it have a preview or beta suffix?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,32 @@
+2.0.0 (In Progress) - Release
+===================
+
+Breaking API Changes
+--------------------
+
+### Active call no longer accessible through the Device object.
+
+`Device.activeCall` is no longer available. Instead,
+the application should keep track of active calls itself, i.e. when accepting an
```
Comment (idelgado) [inline CHANGELOG.md:10]: > Instead, the application should keep a reference to any call that is made using connect or accepted.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,10 +1,32 @@
...
+incoming call or when making an outgoing call.
+
+```ts
+const token = '...';
+const options = { ... };
+const device = new Device(token, options);
+
+await device.register();
+
+const activeCall = await device.connect();
```
Comment (idelgado) [inline CHANGELOG.md:20]: I would just name this `call`.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,10 +1,32 @@
+2.0.0 (In Progress) - Release
```
Comment (idelgado) [inline CHANGELOG.md:1]: When 2.0.0 is released will the changelog for the Pilot and the Release be combined? As a developer upgrading to 2.0.0 from 1.X I would expect all the changes to be in one place.

File: CHANGELOG.md (hunk 2 above)
Comment (mhuynh5757) [reply CHANGELOG.md:23]: It did not, but it was also never released to `npm`, only a GitHub release tag.

File: CHANGELOG.md (hunk 5 above)
Comment (mhuynh5757) [reply CHANGELOG.md:1]: I believe this should be the case, I will talk to Brandon Hawkins about this.

**Pull request merged**